src/gui/newwizard/pages/abstractsetupwizardpage.h

```diff
@@ -11,6 +11,13 @@ class AbstractSetupWizardPage : public QWidget
 public:
     ~AbstractSetupWizardPage() override;
 
+    /**
+     * Check whether the user input appears valid, so that the user can be allowed to proceed through the wizard.
+     * A minimal check for instance is to check whether all required line edits have got data.
+     * In case a page does not have user input to be validated, this function may always return true.
+     */
+    virtual bool inputValidated() = 0;
```

Comment:
maybe just validate?

Reply:
I found that term ambiguous. We just check whether the user input appears valid. This does not mean that it is valid for the server as well, for instance.

Reply:
validateInput();
(Anyhow its just naming so feel free to ignore)